src/test/java/uk/gov/digital/ho/hocs/correspondence/JSONValidate.java

```diff
@@ -144,19 +143,130 @@ public void exceedStringMaxLengths() throws Exception {
                 expectedMessages.add("$.complaint.reporterDetails.applicantEmail: may only be 256 characters long");
                 expectedMessages.add("$.complaint.reporterDetails.applicantPhone: may only be 50 characters long");
                 expectedMessages.add("$.complaint.complaintDetails.applicationSubmittedWhen: may only be 50 characters long");
-                expectedMessages.add("$.complaint.complaintDetails.complaintText: may only be 99999 characters long");
+                expectedMessages.add("$.complaint.complaintDetails.complaintText: may only be 50000 characters long");
```

Comment:
```suggestion
                expectedMessages.add("$.complaint.complaintDetails.complaintText: may only be 50,000 characters long");
```

I'm not sure this validation message makes sense to users. Maybe "Must be *less* than 50,000" would be better?

Reply:
These messages are generated from the json validator library and won't be seen by any end-users. The HOF team are putting their own validation on the form so these messages _shouldn't_ be seen.